consensus/src/marshal/actor.rs

```diff
@@ -129,7 +132,7 @@ pub struct Actor<
     // Last height processed by the application
     last_processed_height: u64,
     // Pending application acknowledgement, if any
-    pending_ack: OptionFuture<PendingAck<B>>,
+    pending_ack: OptionFuture<PendingAck<B, A>>,
```

Comment:
I originally considered migrating to `channels::tracked` (so we could enqueue multiple blocks at once) but opted to take a simpler approach at first to unblock.